test/scenarios.py

```diff
@@ -4090,6 +4090,188 @@
             ),
         },
     ),
+    Scenario(
+        "scenario21-3",
+        "Cancel the directory transfer in flight, then resume transfer. Expect it to be resumed properly",
+        {
+            "ren": ActionList(
+                [
+                    action.ConfigureNetwork(),
+                    action.WaitForAnotherPeer(),
+                    action.NewTransfer("172.20.0.15", ["/tmp/nested/big"]),
+                    action.Wait(
+                        event.Queued(
+                            0,
+                            {
+                                event.File(
+                                    FILES["nested/big/testfile-01"].id,
+                                    "big/testfile-01",
+                                    10485760,
+                                ),
+                                event.File(
+                                    FILES["nested/big/testfile-02"].id,
+                                    "big/testfile-02",
+                                    10485760,
+                                ),
+                            },
+                        )
+                    ),
+                    action.WaitRacy(
+                        [
+                            event.Start(0, FILES["nested/big/testfile-01"].id),
+                            event.Start(0, FILES["nested/big/testfile-02"].id),
+                        ]
+                    ),
+                    action.Wait(event.FinishTransferCanceled(0, True)),
+                    action.WaitForAnotherPeer(),
```

Comment:
`WaitForAnotherPeer` is intended to be at the top of the scenario to ensure the other side has booted up, in here it servers just as an unnecessary delay it seems.

Reply:
Yes, I probably pasted this here by accident. Thanks :+1:

Reply:
Done